components/workspacekit/cmd/rings.go

```diff
@@ -365,6 +501,29 @@ var ring2Cmd = &cobra.Command{
 			return
 		}
 
+		// Now that we're in our new root filesystem, including proc and all, we can load
+		// our seccomp filter, and tell our parent about it.
+		scmpFd, err := seccomp.LoadFilter()
+		if err != nil {
+			log.WithError(err).Error("cannot load seccomp filter - syscall handling would be broken")
+			failed = true
+			return
+		}
+		connf, err := conn.File()
```

Comment:
I don't fully understand why `conn` - a connection build on an Unix socket that is basically a special kernel file (or not...?) - still works after `pivotRoot` :thinking_face:

Reply:
because we already have the FD open. If we wanted to connect to the socket post `pivot_root` that wouldn't work. But because we've done that beforehand, we're fine :)